package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -307,7 +307,18 @@ class HELANAL(AnalysisBase):
     flatten_single_helix : bool, optional
         Whether to flatten results if only one selection is passed.
     split_residue_sequences : bool, optional
-        Wether to split residues into a list of same or consecutive helices.
+        Whether to split residues into a list of same or consecutive helices.
+        If False, the residues will be analysed as a single helix.
+        If True, split residues having gaps into a list of consecutive helices.
+        This will be checked when `select` is having gaps in residue.
+        example-1: if select=['resid 6:50 or resid 100:130 or resid 132:148']
+                   and split_residue_sequences = True
+                   then selection has gaps in the residues.
+                   So Splitting into 3 helices.
+        example-2: if select=['resid 6:50', 'resid 100:130', 'resid 132:148']
+                   and split_residue_sequences = True
+                   then selection has no gaps in the residues.
+                   So Splitting into 1 helix.
```

Comment:
The below might be a more concise summary of this docstring.

```suggestion
        Whether to split the residue sequence into individual helices.
        This keyword only applies if a residue gap is present in the
        AtomGroup generated by a ``select`` string.
        If ``False``, the residues will be analysed as a single helix.
        If ``True``, each group of consecutive residues will be treated as a separate helix.
```

We typically do not put examples in a parameter's documentation; instead, there is usually an examples section laid out by the heading:

```
Examples
----------

```

or you could modify the module docstring as suggested by @PicoCentauri here: https://github.com/MDAnalysis/mdanalysis/blob/32de86f149561b2cea474d098b135c7ffdb5949f/package/MDAnalysis/analysis/helix_analysis.py#L54-L82

However, example 2 is misleading. Even though it's true that the selection is split, it's best to write documentation from the perspective of what a user cares about; they usually only care about how that affects the final result. The resulting analysis still has 3 helices because you've passed in 3 selection strings. Showing an example of `select='resid 6:50 or resid 100:130 or resid 132:148'` and `split_residue_sequences=True|False` would have been more informative.

IMO adding this as an example is optional, though.

Reply:
So, I should only add example - 1 then right? @lilyminium

Reply:
> IMO adding this as an example is optional, though.

so does this means it's fine even if I don't add any example? shall I skip adding it?